src/config.js

```diff
@@ -229,6 +234,17 @@ export default class Config {
     this.workspaceRootFolder = await this.findWorkspaceRoot(this.cwd);
     this.lockfileFolder = this.workspaceRootFolder || this.cwd;
 
+    // using focus in a workspace root is not allowed
+    if (this.focus && (!this.workspaceRootFolder || this.cwd === this.workspaceRootFolder)) {
+      throw new MessageError(this.reporter.lang('workspacesFocusRootCheck'));
+    }
+
+    if (this.focus) {
+      const focusedWorkspaceManifest = await this.findManifest(this.cwd, false);
+      invariant(focusedWorkspaceManifest, `missing manifest at ${this.cwd}`);
+      this.focusedWorkspaceName = focusedWorkspaceManifest.name;
+    }
```

Comment:
You should use `readRootManifest`

Reply:
that hardcodes isRoot to true for normalizeManifest though. Isn't the root in this case the root of the whole project, not the individual workspace that is being focused on? 

Maybe it would make more sense to say readManfiest and pass false for isRoot? Or does it not make a difference for this?